pycyphal/transport/udp/_ip/_endpoint_mapping.py

```diff
@@ -4,47 +4,59 @@
 
 import typing
 import ipaddress
-from pycyphal.transport import MessageDataSpecifier, ServiceDataSpecifier
+
+from numpy import isin
+from pycyphal.transport import DataSpecifier, MessageDataSpecifier, ServiceDataSpecifier
 
 IPAddress = typing.Union[ipaddress.IPv4Address, ipaddress.IPv6Address]
 """
 I wonder why the common base class of IPv4Address and IPv6Address is not public?
 """
 
-IP_ADDRESS_NODE_ID_MASK = 0xFFFF
+MESSAGE_ID_MASK = 2**13-1
 """
-Masks the least significant bits of the IP address (v4/v6) that represent the node-ID.
-Does not apply to multicast group address.
+Masks the least significant bits of the multicast group address (v4/v6) that represent the message-ID.
 """
 
-MULTICAST_GROUP_SUBJECT_ID_MASK = 0xFFFF
+SERVICE_ID_MASK = 0xFFFF
 """
-Masks the least significant bits of the multicast group address (v4/v6) that represent the subject-ID.
+Masks the least significant bits of the multicast group address (v4/v6) that represent the service-ID.
 """
```

Comment:
Would it make sense to have both be `0xFFFF`?

Reply:
It should be "subject-ID", not "message-ID". Per the Specification, subject-IDs are in [0,8192), and service-IDs are in [0,512):

<img width="660" alt="image" src="https://user-images.githubusercontent.com/3298404/198892455-c44ef343-0814-4e29-a04b-9832327fc2ea.png">

The second part of the change is incorrect because the service-ID is not represented in the multicast group address, but rather in the destination port number; see `udp_port_from_data_specifier()`.